site/sfguides/src/streamlit_dashboard/streamlit_dashboard.md

```diff
@@ -0,0 +1,395 @@
+id: streamlit_dashboard  
+summary: Learn how to create an interactive YouTube analytics dashboard using Streamlit within Snowflake Notebooks. This guide covers data generation, visualization, and user interaction to effectively communicate your data insights.  
+categories: featured,streamlit,getting-started,data-science
+environments: web  
+status: Published  
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues  
+tags: Streamlit, Python, Data Visualization
+author: Chanin Nantasenamat
+
+<!-- ------------------------ -->
+# How to Build a YouTube Analytics Dashboard with Streamlit in Snowflake Notebooks
```

Comment:
@sfc-gh-cnantasenamat Please rename this to "Building a YouTube Analytics Dashboard with Streamlit in Snowflake Notebooks"